lang/lang.pt-BR.json

```diff
@@ -191,7 +191,7 @@
             },
             "translate": {
                 "title": "{{REF:bot.name}} - Traduções",
-                "description": "Um agradecimento especial aos tradutores do {{REF:bot.name}} que possibilitaram essa tradução para ser utilizada nessa língua. Se você está interessado em ajudar com a tradução, procure um staff em [support server]({{REF:links.support}})."
+                "description": "Obrigado aos nossos tradutores que possibilitaram que {{REF: bot.name}} fosse usado nos seguintes idiomas. Se você estiver interessado em fornecer uma tradução, entre em contato com a equipe do [servidor de suporte]({{REF:links.support}})."
```

Comment:
Remove the space in `{{REF: bot.name}}` --> `{{REF:bot.name}}`

Reply:
Done.